pymc3/tests/backend_fixtures.py

```diff
@@ -42,13 +42,13 @@ def setUp(self):
 
     def test_append_invalid(self):
         if self.sampler_vars is not None:
-            with self.assertRaises(ValueError):
+            with pytest.raises(ValueError):
```

Comment:
Just a comment - There' a few examples already here that remind me why I prefer Pytest to Nosetests. I just find it a lot easier now to think about testing code.